install/installer/pkg/components/ws-proxy/deployment.go

```diff
@@ -65,7 +65,7 @@ func deployment(ctx *common.RenderContext) ([]runtime.Object, error) {
 		PriorityClassName: common.SystemNodeCritical,
 		Affinity:          common.NodeAffinity(cluster.AffinityLabelWorkspaceServices),
 		TopologySpreadConstraints: []corev1.TopologySpreadConstraint{
-			corev1.TopologySpreadConstraint{
+			{
```

Comment:
Ah, this change was not mine. Just a moment, let me remove it.
cc @andreafalzetti
/hold

Reply:
Updated!
/unhold

Reply:
I was fixing `gofmt` but then another PR was merged to main to fix it, so I rebased and it fixed the job. I thought this would go away automatically after rebase, sorry :)

Reply:
No worries! ✌️